Tree.cs

```diff
@@ -12,27 +12,99 @@ interface ITapable
     {
         bool Tap();
     }
+    enum AgeCatagory
+    {
+        Young,
+        Mature,
+        Elderly,
+        Unknown
+    }
     class Tree
     {
+        //TODO: Make abstract
+        public Tree(int age_days)
+        {
+            Age_Days = age_days;
+        }
+        //AGE:
         protected int _age;
-        protected bool _diseased;
+        protected (int yearStart, AgeCatagory ageCatagory)[] _ageRanges; // Array of tuples :D
+
+        public int Age_Days
+        {
+            get { return _age; }
+            protected set
+            {
+                if (value >= 0) { _age = value; }
+                else {
+                    throw new ArgumentOutOfRangeException("Age_Days", String.Format("{0} is a negative age.", value));
+                }
+            }
+        }
+        public double Age_Years { get { return _age / (double)365; } }
+
+        protected void Increment_Age(int days)
+        {
+            Age_Days += days;
+        }
+        protected void Increment_Age()
+        {
+            Increment_Age(1);
+        }
```

Comment:
Should these be public? Currently no way to increment age from outside class as far as I can see

Reply:
I was gonna add (when we get to it) a daily function, that would use these methods. I don't think theirs any reason to have to manually change ages after creation. The only abnormal bit is the 3x when next to the river but I was gonna build that into the daily function

Reply:
Ok, cos ur busy & we need to add some more pull requests, imma get Ryan to review this & we can add any changes to another pull